README.md

```diff
@@ -32,9 +37,16 @@ const connectionStrings = [
   'mongodb://shard1-secondary1/local'
 ];
 
-const reader = new MongoOplogReader({ redisClient });
+const reader = new MongoOplogReader({ 
+  redisClient,
+  workersPerOplog: 1
```

Comment:
Would `1` invalidate https://github.com/WW-Digital/mongo-oplog-reader/pull/6/files#diff-04c6e90faac2675aa89e2176d2eec7d8R7 ?

Reply:
Ok, thanks for the clarification